post-processing/post_processing.py

```diff
@@ -50,6 +51,10 @@ def run_post_processing(self, config: ConfigHandler):
         # FIXME (#issue #255): have an option to put this into a file (-s / --save flag?)
         print("Selected dataframe:")
         print(self.df[self.mask][config.plot_columns])
+        if self.debug:
+            print("CSV dataframe:")
+            print(self.df[self.mask][config.plot_columns + config.extra_columns])

```

Comment:
Please don't duplicate lines 52-53, just add the `+ config.extra_columns` to the `print` statement in line 53- making sure it doesn't error if `config.extra_columns` doesn't exist...